cmd/lit-af/shell.go

```diff
@@ -223,14 +225,42 @@ func (lc *litAfClient) Ls(textArgs []string) error {
 		return err
 	}
 	if len(cReply.Channels) > 0 {
-		fmt.Fprintf(color.Output, "\t%s\n", lnutil.Header("Channels:"))
+		fmt.Fprintf(color.Output, "\t%s\n", lnutil.Header("Closed Channels:"))
```

Comment:
I think this should just read `Channels` and not `Closed Channels`

Reply:
Will change along with other updates.